Space Invaders game added

singh-anushka opened the pull request:
Fixes [#162](https://github.com/ssitvit/Code-Canvas/issues/162)

singh-anushka commented:
@Shreyaar12 @cleveranu @Dharmesh177 @artimanputra I added the screenshot of the game. Can you please check and merge it with GSSoC23 and the level 2 label?

artimanputra commented:
> @Shreyaar12 @cleveranu @Dharmesh177 @artimanputra I added the screenshot of the game. Can you please check and merge it with GSSoC23 and the level 2 label?

Please share a screen recording of your project

singh-anushka commented:
> > @Shreyaar12 @cleveranu @Dharmesh177 @artimanputra I added the screenshot of the game. Can you please check and merge it with GSSoC23 and the level 2 label?
> 
> Please share a screen recording of your project

I have added the recording of the project along with the rules of the game. Kindly merge it with the main code.

singh-anushka commented:
Please merge this with the main code.
@artimanputra @Dharmesh177 @Shreyaar12 @cleveranu
